mora/app.py

```diff
@@ -24,8 +26,19 @@
 app = flask.Flask(__name__, root_path=distdir, template_folder=templatedir)
 app.cli = cli.group
 
+# Session setup
+app.config.update({
+    'SESSION_TYPE': 'filesystem',
+    'SESSION_PERMANENT': False,
+    'SESSION_FILE_DIR': settings.SESSION_FILE_DIR
+})
```

Comment:
This is the first instance of using actually Flask's configuration in MO, I believe. It would be nice if could move the code towards a more customary use of Flask, where we actually generated the `flask.Flask` object in a method, and used their nice configuration rather than our own JSON file.

Reply:
Absolutely, that would be nice. Let's do that at some point (tm)